docs/ruleset_functions/expected_system_type_from_Table_G3_1_1.md

```diff
@@ -0,0 +1,117 @@
+# expected_system_type_from_Table_G3_1_1
```

Comment:
How does this function related to the does_zone_meet_G3_1_1c?

Reply:
it doesn't really.  The function `get_zone_target_baseline_system` first selects the predominant system type based on HVAC_BAT and Table_G3_1_1(`expected_system_type_from_Table_G3_1_1`), then it selects non-predominant system types for HVAC_BAT > 20,000 ft2 again using Table_G3_1_1 (`expected_system_type_from_Table_G3_1_1`).  Then it goes through the exceptions.  For any zone that meets an exception it changes the expected system type and the string.